bundle_conda.py

```diff
@@ -0,0 +1,295 @@
+"""
+Create napari installers using `constructor`.
+
+It creates a `construct.yaml` file with the needed settings
+and then runs `constructor`.
+
+For more information, see Documentation> Developers> Packaging.
```

Comment:
We might as well put a link in here for convenience?

Reply:
This page does not exist yet so I am unsure how the URL will look.